src/analyses/variable-sensitivity/interval_abstract_value.cpp

```diff
@@ -537,6 +569,17 @@ interval_abstract_valuet::interval_abstract_valuet(
 {
 }
 
+index_range_ptrt interval_abstract_valuet::index_range() const
+{
+  if (is_top() || is_bottom() || interval.is_top() || interval.is_bottom())
+    return std::make_shared<empty_index_ranget>();
+  if (interval.get_lower().id() == ID_min || interval.get_upper().id() == ID_max)
+    return std::make_shared<empty_index_ranget>();
```

Comment:
I get that if the interval is bottom you should get an empty range but I'm not sure I get why this is the case for top or if the interval is open at one end.

Reply:
It doesn't make sense to me either, but this is the existing behaviour.